app/backend/src/app.py

```diff
@@ -69,6 +75,7 @@ def log_unhandled_exception(exc_type, exc_value, exc_traceback):
         # call the default excepthook saved at __excepthook__
         sys.__excepthook__(exc_type, exc_value, exc_traceback)
         return
+    sentry_sdk.capture_exception(exc_value)
```

Comment:
Are you sure exc_value is the right thing to put in here?

Reply:
I thought that was the exception itself. I will check now to see if we might be able to skip this completely, as Sentry itself also generates an unhandled exception hook.

Reply:
Checked, don't need this bit of code, removed it.